scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -33,21 +33,24 @@ task M2 {
   File? cosmic_index
 
   command {
+    if [ -s "$normal_sample_name" -a "$normal_sample_name" != "NO_NORMAL" ]; then
+      normal_command_line="-I ${normal_bam} -normal ${normal_sample_name}"
+    fi
+
     java -Xmx4g -jar ${gatk4_jar} Mutect2 \
     -R ${ref_fasta} \
     -I ${tumor_bam} \
-    -I ${normal_bam} \
     -tumor ${tumor_sample_name} \
-    -normal ${normal_sample_name} \
+    $normal_comand_line \
     ${"--dbsnp " + dbsnp} \
     ${"--cosmic " + cosmic} \
     ${"--normal_panel " + pon} \
     -L ${intervals} \
-    -O ${tumor_sample_name}-vs-${normal_sample_name}.vcf
+    -O "${tumor_sample_name}.vcf"
   }
 
   output {
-    File output_vcf = "${tumor_sample_name}-vs-${normal_sample_name}.vcf"
+    File output_vcf = "${tumor_sample_name}.vcf"
```

Comment:
Done, but it requires a bit of a hack because the cromwell doesn't let you refer to a `String?` in output or input.  Since I couldn't use `File output_vcf = "${tumor_sample_name}{"-vs-" + normal_sample_name}.vcf"` I wrote a brief task to compute this in bash.